### ISSUE TITLE
[Bug] [Integration] [Jira] A configuration error occurs when the integration runs

### ISSUE THREAD

**Issue by usameavci** [port-labs/ocean#271]
#### Ocean Version
🌊 Ocean version: 0.4.7

#### Helm Version
version.BuildInfo{Version:"v3.12.3", GitCommit:"3a31588ad33fe3b89af5a2a54ee1d25bfe6eaa5e", GitTreeState:"clean", GoVersion:"go1.20.7"}

#### K8s Version
1.28
Platform version: eks.6
Kubelet version: v1.28.3-eks-e71965b
AMI: amazon-eks-node-1.28-v20231201

### Integration Version

ghcr.io/port-labs/port-ocean-jira:latest

### Steps to reproduce

Fill secret fields and run, then container logs error.
``` shell
helm upgrade --namespace observability-portio --create-namespace --install ocean-jira port-labs/port-ocean \  
        --set port.clientId="{SECRET}"  \                                    
        --set port.clientSecret="{SECRET}"  \
        --set port.baseUrl="https://api.getport.io"  \
        --set initializePortResources=true  \
        --set integration.identifier="jira"  \
        --set integration.type="jira"  \
        --set integration.eventListener.type="POLLING"  \
        --set integration.config.jiraHost="https://{SECRET}.atlassian.net"  \
        --set integration.secrets.atlassianUserEmail="{SECRET}"  \
        --set integration.secrets.atlassianUserToken="{SECRET}"
```

### What did you expect to see?

I'm waiting to see it work.

### What did you see instead?

[ocean-jira-jira-container.log](https://github.com/port-labs/ocean/files/13696440/ocean-jira-jira-container.log)

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by yairsimantov20** [comment]
Upon reviewing the provided logs, it is evident that the integration encountered an issue while attempting to initialize the necessary resources. 
This problem arose due to conflicts with pre-existing blueprints. Specifically, the error messages indicate that identifiers such as "issue" and "project" were already in use, emphasizing the requirement for unique blueprint identifiers.

```
2023-12-17 13:59:50.731 | ERROR    | Request failed with status code: 409, Error: {"ok":false,"error":"identifier_taken","message":"The identifier \"issue\" is taken, blueprint's identifier should be unique","details":{"identifier":"issue"}}
2023-12-17 13:59:50.743 | ERROR    | Request failed with status code: 409, Error: {"ok":false,"error":"identifier_taken","message":"The identifier \"project\" is taken, blueprint's identifier should be unique","details":{"identifier":"project"}}
2023-12-17 13:59:50.744 | WARNING  | Failed to create resources: {"ok":false,"error":"identifier_taken","message":"The identifier \"project\" is taken, blueprint's identifier should be unique","details":{"identifier":"project"}}. Rolling back changes...
2023-12-17 13:59:50.744 | WARNING  | Failed to create resources: {"ok":false,"error":"identifier_taken","message":"The identifier \"issue\" is taken, blueprint's identifier should be unique","details":{"identifier":"issue"}}. Rolling back changes...
```

When the integration is unable to instantiate the required resources, it defaults to the resource mapping, leading to the occurrence of the error. Additionally, the following error messages suggest that the integration port app configuration is either missing or invalid. It is crucial to configure the port app properly using Port's API.


```
2023-12-17 13:59:51.229 | ERROR    | The integration port app config is empty. Please make sure to configure your port app config using Port's API.
2023-12-17 13:59:51.229 | ERROR    | Invalid port app config found. Please check that the integration has been configured correctly.
```

**Comment by sarpavci** [comment]
Thank you @yairsimantov20. I made a sentry connection before, I think the `issue` identifier was used here. How can I customize these blueprint identifiers for jira?

**Comment by yairsimantov20** [comment]
@usameavci Thank you for letting us know!
It seems that there is a blueprint identifier conflict between the Jira integration and the Sentry integration. I will address this by ensuring that all integration resources are prefixed with the integration name to resolve the conflict.

In the meantime, you can manually create the following [blueprints and app configuration](https://github.com/port-labs/ocean/tree/main/integrations/jira/.port/resources).

**Comment by sarpavci** [comment]
I think I can temporarily solve the problem if I mount the customized files in the resources folder as volume to the target path in the container. I will try to test it today, and then I will write the result.

**Comment by yairsimantov20** [comment]
FYI we do plan on adding the feature you just mentioned as part of the helm chart capabilities

Issue: #232

**Comment by yairsimantov20** [comment]
Closed and moved to #232
